### PR TITLE
Feat support for CSharp/.NET

### PR DESCRIPTION
**Pull request by eldadfux**
## What does this PR do?

Adding support for the new SDK

## Test Plan

N/A

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

#419

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -56,7 +56,7 @@
     },
     "require-dev": {
         "swoole/ide-helper": "4.5.5",
-        "appwrite/sdk-generator": "0.2.3",
+        "appwrite/sdk-generator": "dev-master",
```
Comment (TorstenDittmann) [inline composer.json:59]: You want this for production?

Comment (eldadfux) [reply composer.json:59]: Ohh no! This was just me testing things out :)

**Review by TorstenDittmann — APPROVED** [review]

**Comment by eldadfux** [comment]
Updated code examples, @TorstenDittmann please validate 🙏🏻

**Review by TorstenDittmann — COMMENTED** [review]
lgtm :+1:

Once I validated the file upload, the docs should be fine :)

File: docs/examples/0.6.2/server-dotnet/examples/avatars/get-browser.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+client
+  .SetEndPoint("https://[HOSTNAME_OR_IP]/v1") // Your API Endpoint
+  .SetProject("5df5acd0d48c2") // Your project ID
+  .SetKey("919c2d18fb5d4...a2ae413da83346ad2") // Your secret API key
+;
+
+Avatars avatars = new Avatars(client);
+
+string result = await avatars.GetBrowser("aa");
```
[the remark this answers is not in the document]
Comment (eldadfux) [reply docs/examples/0.6.2/server-dotnet/examples/avatars/get-browser.md:13]: Seems like a yes: https://github.com/appwrite/appwrite/blob/d6ad0779b332ff1a867fc50182c5caf1d1562cf5/docs/lists/clients.json#L4

[the remark this answers is not in the document]
Comment (TorstenDittmann) [reply docs/examples/0.6.2/server-dotnet/examples/avatars/get-browser.md:13]: Oh my, that list 😅 hopefully nobody's using Google Earth to browse the web.

Good to know :+1:

[the remark this answers is not in the document]
Comment (eldadfux) [reply docs/examples/0.6.2/server-dotnet/examples/avatars/get-browser.md:13]: Yeh, We need to find an elegant way to combine it in the docs: https://appwrite.io/docs/client/avatars#getBrowser

**Comment by eldadfux** [comment]
I have added a `dev` attribute to the platform list. The new attribute will help hide the SDK from the homepage until the release announcement is made.

**Pull request merged**